openverse_catalog/dags/common/loader/reporting.py

```diff
@@ -28,7 +32,11 @@ def humanize_time_duration(seconds):
     return ", ".join(parts)
 
 
-def report_completion(provider_name, duration, record_counts_by_media_type):
+def report_completion(
+    provider_name: str,
+    duration: float | str | None,
```

Comment:
When is `duration` a string? (Now that pickling is turned on)

Reply:
Hmm, it shouldn't be, but I thought I ran into a case where it was :thinking: certainly couldn't hurt to retain the logic that accounts for it being a non-float type...